x-pack/plugin/snapshot-repo-test-kit/src/test/java/org/elasticsearch/repositories/blobstore/testkit/analyze/AbstractRepositoryAnalysisRestTestCase.java

```diff
@@ -29,7 +29,7 @@ public void testRepositoryAnalysis() throws Exception {
         final Request request = new Request(HttpPost.METHOD_NAME, "/_snapshot/" + repository + "/_analyze");
         request.addParameter("blob_count", "10");
         request.addParameter("concurrency", "4");
-        request.addParameter("max_blob_size", "1mb");
+        request.addParameter("max_blob_size", randomFrom("1mb", "10mb"));
```

Comment:
is it worth increasing the range to 15 or 20? To get more than 2 chunks?

Reply:
2 is enough, I can't really see how we could get this right with 2 chunks but wrong with more.